### DISCUSSION TITLE
FireFly-desktop 1.7.0 not running on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Gonzalei** [iotaledger/firefly#5708] [Help & Support]
Hi, I have Ubuntu 22.04 and I run firefly-desktop-1.7.0.AppImage  but the window app is blank. Don show any thing.
Can someone help me to fix the problem.
Thank you in advance
Ignacio
![Screenshot from 2023-01-31 11-33-58](https://user-images.githubusercontent.com/47270232/215738910-2889dcff-5e25-4a31-9aba-697fec347b10.png)

**Comment by StarwoodSystems** [comment]
I had the same issue - blank page on trying to open Firefly and resolved it by moving the app to a partition with more space.  Check that you have sufficient space on the partition that you are using. It requires several gigs.
